guides/v2.2/config-guide/bootstrap/mage-profiler.md

```diff
@@ -54,7 +54,12 @@ To enable or disable the profiler:
 1. Log in to your Magento server.
 1. Change to your Magento installation directory.
 1. As the Magento file system owner, enter the following command to configure the profiler:
-
+   Please ensure you are running the Magento application in [developer mode]({{ page.baseurl }}/config-guide/bootstrap/magento-modes.html).
```

Comment:
Hi @konarshankar07. Could you clarify, please, the necessity of enabling the developer mode for turning on the profiler? The profiler works well in all modes (including production). Also, very often it might be useful to use profiler in production mode in order to test the performance on a production-ready system. 

Thank you

Reply:
Hello @rogyar ,
Yes, You are right. Its my mistake that I didn't checked the issue mention in the ticket. But still try to reach the person who opened the ticket and try to clarify what issue is he getting
Thanks